### DISCUSSION TITLE
Kaspersky Internet Security

### DISCUSSION THREAD

**Discussion by s58122202001** [elrumo/macOS_Big_Sur_icons_replacements#1272] [Icon request]
I would like to request an icon for [Kaspersky Internet Security](https://www.kaspersky.com/mac-security).

Thank you.

**Comment by johnlamox** [comment]
Done! Waiting for Approval here: https://macosicons.com/u/johnlamox

![Kaspersky Internet Security](https://user-images.githubusercontent.com/57749200/129879797-4a580316-aef9-4bd7-9e80-1661d8df38aa.png)
